pages/min-side/[[...slug]].tsx

```diff
@@ -106,7 +106,6 @@ const Home: LayoutPage<{ data: any }> = ({ data }) => {
       filterYear = router.query.slug[1];
     }
   }
-  console.log(filterYear);
```

Comment:
Console log :P

Reply:
Yep, but the change is that I'm removing it :P

Reply:
My bad 🤦‍♀️